tests/integration/utils/cluster.py

```diff
@@ -66,6 +67,12 @@ async def start_schema_registry_cluster(
             logfile = stack.enter_context(open(log_path, "w"))
             errfile = stack.enter_context(open(error_path, "w"))
 
+            client_auth = (
+                config.server_tls_client_auth.value
+                if isinstance(config.server_tls_client_auth, enum.Enum)
```

Comment:
Is this really needed?

Reply:
its good to keep it imo, as it normalizes the value just in case if it is not enum rather its a raw string. doesn't hurt